Plugins/Creature/Creature.cpp

```diff
@@ -51,10 +51,12 @@ static bool s_bCriticalMultiplierHooksInitialized = false;
 static bool s_bCriticalRangeHooksInitialized = false;
 static bool s_bResolveAttackRollHookInitialized = false;
 static bool s_bResolveInitiativeRollHookInitialized = false;
+static bool s_bGetIsClassAvailableHookInitialized = false;
 static Hooks::Hook s_GetClassLevelHook = nullptr;
 
 static std::unordered_map<uint8_t, std::unordered_map<ObjectID, int16_t>> s_RollModifier;
 static std::unordered_map<ObjectID, bool> s_ParryAllAttacks;
+static std::unordered_map<OBJECT_ID, uint> s_MulticlassLimits;
```

Comment:
Why an unordered_map instead of using NWNX's POS? With POS you can have it persist to the bic too